test/e2e-common.sh

```diff
@@ -41,7 +41,7 @@ function test_setup() {
 
   # Bringing up controllers.
   echo ">> Bringing up controller"
-  ko apply -f config/ || return 1
+  ko apply -f third_party/gateway-api/ || return 1
```

Comment:
This doesn't look like it's bringing up the controllers anymore. I don't know if we need both applied. In particular, line 56 says "Deploy Gateway API resources".

Reply:
Good catch wrt ln 56. That bit already is in the istio specific e2e test file (kind-e2e-istio.sh), as it should. I'll remove it.